app/assets/stylesheets/header-footer-only.scss

```diff
@@ -12,7 +12,6 @@ $govuk-use-legacy-palette: false;
 @import "css3";
 @import "device-pixels";
 @import "measurements";
-@import "typography";
```

Comment:
Does this file need `govuk_frontend_support` now that `typography` has been removed?

Reply:
It already imports `govuk_frontend_support` at line 4 (unless you're implying it _doesn't_ need it?) 🤔

Reply:
Ah, sorry, missed that, thanks.